docs/supply-chain-tools/manufacturing/manufacturer-enablement-guide.md

```diff
@@ -94,7 +94,7 @@ Table 3.	Linux* OS-based Manufacturer Toolkit
 |    Connectivity        |    Two Ethernet networks: one to connect   to the IoT device and a separate network to connect to the backend to provide   access to voucher data and the voucher extension operation.                                                                                                                                                                                            |
 |    Secure Key Fobs     |    YubiKey* security key (see <https://www.yubico.com/products/yubikey-hardware/>)                                                                                                                                                                                                                                                                                                  |
 |    Operating System    |    Ubuntu* OS version 16.x or 18.x                                                                                                                                                                                                                                                                                                                                                |
-|    Software            |    Java\* Runtime Environment (JRE) software version 11 </p>    Oracle\* software (<http://www.oracle.com/technetwork/java/javase/downloads/jre8-downloads-2133155.html>) </p>   or  </p> OpenJDK\* (<http://openjdk.java.net/install/index.html>) </p>  · Apache Tomcat\* (<http://tomcat.apache.org/>)   software version 8 or later </p>   · MariaDB* (<https://mariadb.org/>)  server version 10.4 or later </p>   |
+|    Software            |    Java\* Runtime Environment (JRE) software version 11 </p>    Oracle\* software (https://www.oracle.com/java/technologies/javase/jdk11-archive-downloads.html) </p>   or  </p> OpenJDK\* (<http://openjdk.java.net/install/index.html>) </p>  · Apache Tomcat\* (<http://tomcat.apache.org/>)   software version 8 or later </p>   · MariaDB* (<https://mariadb.org/>)  server version 10.4 or later </p>   |
```

Comment:
We should be using reference to only OpenJDK here. You may remove Oracle JRE link.

Reply:
Done